### ISSUE TITLE
Import new Event with specific EventID

### ISSUE THREAD

**Issue by swebster96** [MISP/PyMISP#862]
The EventId can be used in an API call or within PyMISP utility to delete or update a specific event.

Is there a MISP configuration to enable so when adding a new event, a specific EventId can be used?

`{
	"Event": {
		"id": "222222",
		"orgc_id": "2",
		"org_id": "1",
		"date": "2022-09-22",
		"threat_level_id": "3",
		"info": "test event",
		"published": false,
		"attribute_count": "0",
		"analysis": "2",
		"timestamp": "1663876708",
		"distribution": "0",
		"proposal_email_lock": false,
		"locked": true,
		"publish_timestamp": "0",
		"sharing_group_id": "0",
		"disable_correlation": false,
		"extends_uuid": "",
		"protected": null,
	}
}`

rather than MISP autogenerating the next available event number.

**Comment by Rafiot** [comment]
No, there is no way to do that, The Event ID is auto incremented ad the DB level.

I'd recommend you to use UUIDs instead.

**Comment by swebster96** [comment]
Thanks for this reply.  We will need to architect a solution to store the UUID in the database associated with the ID for sync/delta needs.



Scott
